test/core/Constant.t.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier:	AGPL-3.0
+
+// those tests should be run with -vv so correct gas estimates are shown
```

Comment:
There are no gas estimates in these tests? Copy-paste leftover?

Reply:
fixed in 962779950c4407e50d1a2c9882e0f1ff6bfb4d56